apps/sertra/package.json

```diff
@@ -17,7 +17,7 @@
     "next": "12.1.6",
     "react": "18.1.0",
     "react-dom": "18.1.0",
-    "react-hook-form": "^7.32.2",
+    "react-hook-form": "^7.34.2",
```

Comment:
The commit doesn't include a updated `yarn.lock` file. You should run `yarn install`.